src/main/java/org/opentripplanner/api/model/ApiStopShort.java

```diff
@@ -2,15 +2,16 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import org.opentripplanner.util.I18NString;
 
 public class ApiStopShort {
 
     public String id;
     public String code;
-    public String name;
+    public I18NString name;
```

Comment:
These should be String, and we should do locale-aware mapping.

Reply:
Fixed.